java/private/library.bzl

```diff
@@ -37,7 +37,7 @@ def create_lint_tests(name, **kwargs):
         )
 
     spotbugs = get_lint_config("java-spotbugs", tags)
-    if spotbugs != None:
+    if spotbugs != None and not native.existing_rule("%s-spotbuts" % name):
```

Comment:
```suggestion
    if spotbugs != None and not native.existing_rule("%s-spotbugs" % name):
```

Reply:
Would it make more sense to use a `maybe` rather than hard-coding its impl here in ways which could contain typos?

Reply:
`maybe()` is a repository rule, and this isn't loading a repository. Good thought though.

Reply:
There's actually nothing repository-rule specific about `maybe`, it works everywhere!